### ISSUE TITLE
Wrong gnt-cluster search-tags example in man page

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#115]
Originally reported of Google Code with ID 59.

```
<b>What steps will reproduce the problem?</b>
1. man gnt-cluster
2. go to the search-tags section
3. try the example "gnt-cluster search time"
4. note it should be "gnt-cluster search-tags time"

<b>What is the expected output? What do you see instead?</b>
actual: usage message
expected: no output

<b>What version of the product are you using? On what operating system?</b>
Ganeti 2.0
Ubuntu Linux

<b>Please provide any additional information below.</b>


```
Originally added on 2009-06-11 13:58:36 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in commit 2f49d1d2 and was released in 2.0.1.
```
Originally added on 2009-08-05 08:49:34 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
